lhotse/recipes/timit.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Xiaomi Corporation (Author: Mingshuang Luo)
+# Apache 2.0
+
+from collections import defaultdict
+
+import os
+import glob
+import json
+import logging
+import shutil
+import tarfile
+import string
+from pathlib import Path
+from tqdm.auto import tqdm
+from typing import Dict, Optional, Union
+from concurrent.futures.thread import ThreadPoolExecutor
+
+from lhotse import validate_recordings_and_supervisions
+from lhotse.audio import Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, urlretrieve_progress
+
+
+def download_and_unzip(
+        target_dir: Pathlike = '.',
+        force_download: Optional[bool] = False,
```

Comment:
Why are `force_download` and `base_url` `Optional`?
What if `Optional` is removed?

Reply:
if the timit zip exits, we needn't to download. Optional in here means that we can download or not download. And the base_url is not unique.

Reply:
`Optional` here means you can pass `None` to the argument. Obviously, `base_url` cannot be `None`.

Reply:
Yeah the types should be plain bool and str

Reply:
```suggestion
        force_download: bool = False,
```